brocade-vdx/config.json

```diff
@@ -0,0 +1,15 @@
+{
+     "name": "Brocade VDX SKU",
+     "rules": [
+         {
+             "regex": "Network Operating System",
```

Comment:
I'm worried this is a little too light of a definition. This may match plenty of other non-brocade switches as well.